src/main/java/org/usfirst/frc4048/common/swervev2/type/SparkMaxSwerveModule.java

```diff
@@ -0,0 +1,12 @@
+package org.usfirst.frc4048.common.swervev2.type;
+
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import org.usfirst.frc4048.common.swervev2.components.EncodedSwerveSparkMax;
+import org.usfirst.frc4048.common.util.Gain;
+import org.usfirst.frc4048.common.util.PID;
+
+public class SparkMaxSwerveModule extends GenericSwerveModule {
```

Comment:
The value of this class is limited since the base class can do everything it needs to generically? Maybe we don't need it?